utils/release_helper.py

```diff
@@ -101,88 +235,137 @@ def init_gitlab(self):
             exit(2)
 
-        if self.project.attributes["namespace"]["path"] != project_root_namespace:
+        if (
+            self._gitlab_project.attributes["namespace"]["path"]
+            != self.ci_project_root_namespace
+        ):
             logger.fatal(
-                f"project_root_namespace ({ project_root_namespace }) does not match namespace of Project ({self.project.attributes['namespace']['path']}) "
+                f"project_root_namespace ({ self.ci_project_root_namespace }) does not match namespace of Project ({self._gitlab_project.attributes['namespace']['path']}) "
             )
             logger.error("You might want to: unset CI_PROJECT_ID")
             exit(2)
+        return self._gitlab_project
 
+    @property
+    def ci_project_id(self):
+        if hasattr(self, "_ci_project_id"):
+            return self._ci_project_id
+        if os.environ.get("CI_PROJECT_ID"):
+            self._ci_project_id = os.environ.get("CI_PROJECT_ID")
+            logger.info(f"Using ci_project_id: {self.ci_project_id} ")
+        else:
+            logger.error("No Project given. choose one:")
+            for project in self.gl.projects.list(search="specter-desktop"):
+                logger.info(
+                    f"     export CI_PROJECT_ID={project.id}  # {project.name_with_namespace}"
+                )
+                if project.name_with_namespace.startswith("cryptoadvance"):
+                    self._ci_project_id = project.id
+            logger.warn("{self._ci_project_id} has been chosen as self._ci_project_id")
+        return self._ci_project_id
+
+    @property
+    def github_project(self):
+        if hasattr(self, "_github_project"):
+            return self._github_project
+        self._github_project = f"{self.ci_project_root_namespace}/specter-desktop"
+        logger.info(f"Using github_project: {self._github_project}")
+        return self._github_project
+
+    @property
+    def ci_commit_tag(self):
+        if hasattr(self, "_ci_commit_tag"):
+            return self._ci_commit_tag
         if os.environ.get("CI_COMMIT_TAG"):
-            self.tag = os.environ.get("CI_COMMIT_TAG")
+            self._ci_commit_tag = os.environ.get("CI_COMMIT_TAG")
         else:
             raise Exception("no tag given ( export CI_COMMIT_TAG=v0.0.0.0-pre13 )")
-        logger.info(f"Using tag: {self.tag}")
+        logger.info(f"Using tag: {self._ci_commit_tag}")
+        return self._ci_commit_tag
+
+    @property
+    def ci_project_root_namespace(self):
+        if hasattr(self, "_ci_project_root_namespace"):
+            return self._ci_project_root_namespace
+        if os.environ.get("CI_PROJECT_ROOT_NAMESPACE"):
+            self._ci_project_root_namespace = os.environ.get(
+                "CI_PROJECT_ROOT_NAMESPACE"
+            )
+            logger.info(
+                f"Using project_root_namespace: {project_root_namespace} ( export CI_PROJECT_ROOT_NAMESPACE={project_root_namespace} )"
```

Comment:
`project_root_namespace` is not defined!

Reply:
fixed in [b8a3a7c](https://github.com/cryptoadvance/specter-desktop/pull/2446/commits/b8a3a7ce4ef1e1dfedd24e856d765754f1d5843f)